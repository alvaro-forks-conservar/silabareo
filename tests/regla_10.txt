adquirir
podzol
adjetivo
adlÃ¡tares
administrar
adnato
adviento
addenda
adyacente