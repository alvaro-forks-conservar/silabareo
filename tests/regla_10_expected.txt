ad-qui-rir
pod-zol
ad-je-ti-vo
ad-lÃ¡-ta-res
ad-mi-nis-trar
ad-na-to
ad-vien-to
ad-den-da
ad-ya-cen-te